### DISCUSSION TITLE
Recording & events on/off based on Google calendar

### DISCUSSION THREAD

**Discussion by McPaHa** [blakeblackshear/frigate#2986] [Ideas]
Hi!

I've a summer cottage where occasionally visited other people than me. These visits are in the events at Google calendar. Calendar is empty "normally" but there is meeting event if there is somebody.

Would be very nice & clever if I could automatically toggle recording & events & snapshots & detection of during time when there is calendar event. Is this a good idea for somebody more clever than me?

I'm running frigate with docker at ubuntu based system.

**Comment by NickM-27** [comment]
This is definitely doable, just need something that uses calendar API and sends to frigate mqtt. Could be written in python fairly simply

**Comment by McPaHa** [comment]
I am using Home Assistant (also with docker) but I like to separate Frigate - not using as addon with HA. That is the reason I'm asking this. Otherwise this probably (inside of HA) could be done f.e. with NodeRed.

I've no skills to do it... Just giving an idea and hoping & waiting someby wiser than me do this because probably he/she also needs this with community.

**Comment by blakeblackshear** [comment]
You should install the [frigate integration](https://github.com/blakeblackshear/frigate-hass-integration) (not the same thing as an addon) to get switches in home assistant for frigate. Then you can use the [google calendar integration](https://www.home-assistant.io/integrations/google/) to automate frigate however you want.

**Reply by McPaHa** [reply]
All right... I have frigate integration but haven't though this so deeply... need to try... thx @blakeblackshear
